cpp/include/cudf/detail/null_mask.hpp

```diff
@@ -53,7 +53,7 @@ void set_null_mask(bitmask_type *bitmask,
  * @param[in] stream CUDA stream used for device memory operations and kernel launches.
  */
 std::vector<size_type> segmented_count_set_bits(bitmask_type const *bitmask,
-                                                std::vector<size_type> const &indices,
+                                                host_span<size_type const> indices,
```

Comment:
I assume the `host_span` being passed by value here now is not a performance concern since it's much smaller than a vector (just a wrapper around a raw pointer)? Perhaps more for my edification than anything else: is there any particular reason to prefer not using the reference in this case?

Reply:
I think Vukasin covered this in his Better Code presentation on spans, but in general one of the points of views is you pass by value, though I don't remember why. It's just a pointer and a size.  We don't pass column_view by value because they are not true views. There is an open issue for that one...